### PR TITLE
Add Python 3.9 Support

### PR DESCRIPTION
**Pull request by jan-janssen**
It should be available now https://github.com/conda-forge/stdlib-list-feedstock/pull/13

Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by jan-janssen** [comment]
@conda-forge-admin, please rerender

**Comment by CJ-Wright** [comment]
This looks great! Does `stdlib-list` have tight python version constraints?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -9,19 +9,18 @@ source:
...
-  number: 0
+  number: 1
   noarch: python
   script: {{ PYTHON }} -m pip install . --no-deps -vv
   entry_points:
     - depfinder = depfinder.cli:cli
 
 requirements:
   host:
-    # Python version is limited by stdlib-list.
```
Comment (ocefpaf) [inline recipe/meta.yaml:20]: Being able to run it on Python 3.9 does not mean it can inspect Python 3.9 code without the corresponding `stdlib-list`.

Comment (CJ-Wright) [reply recipe/meta.yaml:20]: Right, I'm hoping that `stdlib-list`'s version constraints will actually make certain versions of python (eg 3.10) unsolvable.

**Comment by jan-janssen** [comment]
Ok, then I close this pull request for now.

**Pull request closed without merging**